### PR TITLE
2.1.6 G5 Header

### PR DESCRIPTION
**Pull request by thescubageek**
One more time.... getting the G5 Header to the latest version (4.3.2) and properly precompiled....

QA Test Site: https://g5-cms-1tbo672w-qa-test-beds.herokuapp.com/
PT: https://www.pivotaltracker.com/n/projects/877197/stories/101630312

### CODE REVIEW

**Comment by bbauer** [comment]
:+1: when :green_heart:

**Comment by thescubageek** [comment]
Specs pass but the actual App fails with a `Uncaught ReferenceError: jQuery is not defined` error when clicking on the toggle menu.

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -19,12 +19,16 @@
...
   <body class="<%= user_class %>">
-    <%# vendor.js needs to be above g5 header since it fulfills its jquery requirement 
-        eventually this needs to be fixed in the g5 header gem possibly wiht a content for block
-        or create an ember addon %>
-    <script src="/assets/vendor.js"></script>
-    <%= g5_header %>
+    <div id='g5_header_container'></div>
     <%= yield %>
+    <script src="/assets/vendor.js"></script>
+    <script>
```
Comment (bbauer) [inline app/views/layouts/application.html.erb]: Nice, yeah if this works, go for it.

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 require "resque/server"
 
 G5CMS::Application.routes.draw do
+  mount G5Header::Engine => '/g5_header'
```
Comment (bbauer) [inline config/routes.rb]: I don't think we need to mount the g5_header route? That's for centralized apps only...

Comment (thescubageek) [inline config/routes.rb]: The jQuery injection will fail if the route isn't mounted:
```
      jQuery.get('/g5_header/render', function(data) {
      jQuery('#g5_header_container').append(data.markup);
      jQuery('head').append(data.style);
      jQuery('body').append(data.javascript);
    });
```

Comment (bbauer) [inline config/routes.rb]: Ah yep!

**Comment by bbauer** [comment]
:+1:

**Comment by thescubageek** [comment]
Merged into 2.1.6-merge: https://github.com/g5search/g5-content-management-system/pull/825

Thanks @bbauer and @jonnysamps !

**Pull request merged**